### PR TITLE
pkp/orcidProfile#232 Display Orcid Icon only for authenticated users

### PR DESCRIPTION
**Pull request by withanage**
### Issue
- https://github.com/pkp/orcidProfile/issues/232

### CODE REVIEW

**Comment by withanage** [comment]
Hi @asmecher ,
This PR  backports  https://github.com/pkp/orcidProfile/issues/232, which was included in [main branch](https://github.com/pkp/ojs/pull/3758)

Would be very useful to have this shipped in 3.3.0-15

**Comment by asmecher** [comment]
Merged, thanks!

File: templates/frontend/objects/article_details.tpl [hunk 1]
```diff
@@ -120,6 +120,9 @@
 							{if $author->getData('orcid')}
 								<span class="orcid">
 									{$orcidIcon}
```
Comment (mhvezina) [inline templates/frontend/objects/article_details.tpl:122]: Shouldn't this line be removed? If not the icon will likely appear twice...

Comment (withanage) [reply templates/frontend/objects/article_details.tpl:122]: @mhvezina  thanks for  your review. 

Actullay, no.    AcessToken is only checked for displaying the orcidLogo. Here is the current  stable-3_3_0  branch.

https://github.com/pkp/ojs/blob/stable-3_3_0/templates/frontend/objects/article_details.tpl#L120

Comment (mhvezina) [reply templates/frontend/objects/article_details.tpl:122]: I meant this line : https://github.com/pkp/ojs/pull/3865/files#diff-959b67b46d8905ed019270b42e656fe1286d6747716787b4681b5de6710c1eaeL122  But indeed now I see the line has been removed in https://github.com/pkp/ojs/blob/stable-3_3_0/templates/frontend/objects/article_details.tpl#LL121C23-L121C23 Thanks.

**Pull request merged**